hkmc2/shared/src/test/mlscript/backlog/UCS.mls

```diff
@@ -159,34 +159,6 @@ fun parse(text: Str): [Int, Str] =
 
 // ——— ——— ———
 
-class
-  Tuple(a, b, c)
-  Ident(n)
-
-fun foo(tree) =
-  fun go(x) = ???
-  if tree is
-    Tuple(category, choice, function) and
-      category is Ident(categoryName) and
-        function is Ident(functionName) and
-          choice is
-            Tuple(elements) then Some(go(elements))
-            other then Some(go(other))
-        else
-          console.warn("expect the choice to be a bracketed term but found " + choice)
-          None
-      else
-        console.warn("expect a the category to be an identifier but found " + category)
-        None
-    else
-      console.warn("expect the definition to be a tuple but found " + tree)
-      None
-//│ ╔══[ERROR] mismatched arity: expect 3, found 1
-//│ ║  l.173: 	            Tuple(elements) then Some(go(elements))
-//│ ╙──       	            ^^^^^^^^^^^^^^^
-
-// ——— ——— ———
-
```

Comment:
Note that I already changed this test in the other branch. Please don't mess the conflict resolution.

Reply:
Sure. I just made sure that this file is not in the change list.